core/src/main/resources/reference.conf

```diff
@@ -184,11 +194,17 @@ system {
     }
 
     gcs {
-      parallelism = 10
+      # `gcs.parallelism` governs GCS *batch operation* parallelism. Non-batch GCS operations like 'write' or
+      # 'read lines' are processed as nio commands whose parallelism is governed by `nio.parallelism` below.
+      parallelism = 5
+      # Batches of non-zero size will be sent to Google with at most `max-batch-size` commands and not having
+      # more than `max-batch-timespan` between when the first and last command was batched.
+      max-batch-size = 20
+      max-batch-timespan = 2 seconds
```

Comment:
New config parameters for previously hardcoded values. These used to be hardcoded to 100 and 5 seconds respectively.